test/t_web/channels/feed_channel_test.exs

```diff
@@ -92,15 +92,35 @@ defmodule TWeb.FeedChannelTest do
                join(socket, "feed:" <> me.id, %{"mode" => "normal"})
 
       %{"voicemail" => voicemail_m5} = Enum.at(matches, 0)
+      %{id: me_id} = me
+      %{id: p5_id} = p5
 
-      assert [
-               %{
-                 id: _id,
-                 inserted_at: _inserted_at,
-                 s3_key: "23f442c7-e610-4aa9-ad4c-7795bb568c4e",
-                 url: _url
-               }
-             ] = voicemail_m5
+      assert %{
+               # messages from mate (= where mate is caller)
+               ^p5_id => [
```

Comment:
or should it be the other way around, messages under mate's id would be sent to them, not sent by them like it is in this test?

Reply:
I would say that voicemail by one party "overwrites" (similar to slots / contact) previous voicemail by other party -> only one list of voicemail will be returned at any time to each user with a `caller_id` for this list

Reply:
how should the payload look like?

Reply:
probably sth like this:

```suggestion
               %{
        "voicemail" => %{
          "caller_id" => "0000017c-14c7-9745-0242-ac1100020000",
          "voicemail" => [
            %{
              id: _id,
              inserted_at: _inserted_at,
              s3_key: "23f442c7-e610-4aa9-ad4c-7795bb568c4e",
              url: _url
            },
            %{
              id: _id,
              inserted_at: _inserted_at,
              s3_key: "23f442c7-e610-4aa9-ad4c-7795bb568c4e",
              url: _url
            }
          ]
        }
      }
```

Reply:
```
"voicemail" => %{
          "caller_id" => "0000017c-14c7-9745-0242-ac1100020000",
          "messages" => [
```

?

Reply:
yeah, this is better 👍